### PR TITLE
fix: React 19 bool prop to attr value bug

### PR DESCRIPTION
**Pull request by luwes**
fix https://github.com/muxinc/elements/issues/1015

### CODE REVIEW

File: packages/mux-player-react/src/common/utils.ts [hunk 1]
```diff
@@ -11,6 +22,7 @@ const ReactPropToAttrNameMap = {
   playsInline: 'playsinline',
   autoPlay: 'autoplay',
   playbackRate: 'playbackrate',
+  defaultMuted: 'muted',
```
Comment (cjpillsbury) [inline packages/mux-player-react/src/common/utils.ts:25]: issue(blocking): This is inaccurate. `muted` needs to apply dynamically over the life of the react player instance (e.g. if someone wants to programmatically toggle `muted` via props in their react app. There is also no attribute called `defaultMuted`. The `muted` HTML attribute will map to the `defaultMuted` HTML element property. Curious why the change was made here.

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
One callout but this general approach should be fine for simple values like booleans in React 19 (with backwards compat). 👌

**Review by cjpillsbury — APPROVED** [review]
🚀

File: packages/mux-player-react/src/common/utils.ts (hunk 1 above)
Comment (cjpillsbury) [reply packages/mux-player-react/src/common/utils.ts]: > confusing stuff.
💯

**Comment by arcataroger** [comment]
❤️

**Pull request merged**